server/common/rest.py

```diff
@@ -159,13 +160,21 @@ def annotations_put_fbs_helper(data_adaptor, fbs):
     annotations.write_labels(new_label_df, data_adaptor)
 
 
+def inflate(data):
+    # https://docs.python.org/3/library/zlib.html#zlib.decompressobj
+    decompress = zlib.decompressobj()
+    decompressed_data = decompress.decompress(data)
```

Comment:
Do you get anything out of doing it this way rather than just calling `zlib.decompress`? The data is all read into memory at once anyway by `get_data`.

Reply:
Ooh that's a really great point! I totally missed the "stream" part of using `zlib.decompressobj()` lol Just updated to use `zlib.decompress()`! Thank you!

Reply:
:+1: